tests/filestore-v2.10-wrong-direction/test.yaml

```diff
@@ -0,0 +1,29 @@
+pcap: ../filestore-filecontainer-http/filecontainer-http.pcap
+
+requires:
+  features:
+    - HAVE_NSS
+    - MAGIC
+  files:
+    - src/output-filestore.c
```

Comment:
Please do not put filenames because we will move them to rust one day like transforms ;-)

Put min-version: 8 rather ;-)

Reply:
Good idea ! Done and pushed on this PR. Let me know if you want me to open another one.